src/main/kotlin/racingcar/view/ResultView.kt

```diff
@@ -0,0 +1,39 @@
+package racingcar.view
+
+import racingcar.domain.car.CarName
+import racingcar.domain.car.CarRacingResult
+
+class ResultView {
+
+    fun show(result: List<CarRacingResult>) {
+        val resultString = StringBuilder()
+
+        resultString.append(makeProgressString(result))
+
+        val winners = findWinners(result.last()).joinToString(", ")
+        resultString.append("${winners}가 최종 우승했습니다.")
+
+        println(resultString)
```

Comment:
```suggestion
        val winners = findWinners(result.last()).joinToString(", ")
        val resultString = buildString {
            append(makeProgressString(result))
            append("${winners}가 최종 우승했습니다.")
        }
        println(resultString)
```
코틀린에서는 StringBuilder 대신 [buildString](https://kotlinlang.org/api/latest/jvm/stdlib/kotlin.text/build-string.html) 을 자주 활용하곤 합니다!